examples/google/google-pubsub/src/main/java/org/apache/camel/example/MyRouteBuilder.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.example;
+
+import org.apache.camel.builder.endpoint.EndpointRouteBuilder;
+
+/**
+ * To use the endpoint DSL then we must extend EndpointRouteBuilder instead of RouteBuilder
+ */
+public class MyRouteBuilder extends EndpointRouteBuilder {
+
+    @Override
+    public void configure() {
+        from(googlePubsub("{{projectId}}:rhoc-test-sub").serviceAccountKey("{{serviceAccountKey}}")).log("The content is ${body}");
+
+        from(timer("tick").period(1000)).setBody(constant("Event Test"))
+                .to(googlePubsub("{{projectId}}:rhoc-test").serviceAccountKey("{{serviceAccountKey}}"));
```

Comment:
Rhoc-test should be configurable i guess

Reply:
thanks - updated